woocommerce/woocommerce-ios
# [Manual Taxes M3] Events Tracking.

Closes: #10785 


## Description

With this PR we track the events regarding Manual Taxes M3.

## Testing instructions


With a tax rate stored (see here https://github.com/woocommerce/woocommerce-ios/pull/10680 for instructions)

1. Go to Orders
2. Tap on + to create a new order
3. Wait until the tax rate is applied
4. Tap on Edit Tax Rate Setting. Event should be tracked:

`🔵 Tracked tax_rate_auto_tax_rate_bottom_sheet_displayed, properties: [AnyHashable("is_wpcom_store"): true, AnyHashable("plan"): "business-bundle", AnyHashable("site_url"): "https://americanwootester.wpcomstaging.com", AnyHashable("was_ecommerce_trial"): false, AnyHashable("blog_id"): 214354650]`

5. On that sheet, tap on Set a new tax rate for this order. Event should be tracked:

`🔵 Tracked tax_rate_auto_tax_rate_set_new_rate_for_order_tapped, properties: [AnyHashable("plan"): "business-bundle", AnyHashable("site_url"): "https://americanwootester.wpcomstaging.com", AnyHashable("was_ecommerce_trial"): false, AnyHashable("is_wpcom_store"): true, AnyHashable("blog_id"): 214354650]`

7. Tap on Cancel. Tap on Edit Tax Rate Setting again.
8. Tap on Clear address and stop using this rate. Event should be tracked:

`🔵 Tracked tax_rate_auto_tax_rate_clear_address_tapped, properties: [AnyHashable("plan"): "business-bundle", AnyHashable("site_url"): "https://americanwootester.wpcomstaging.com", AnyHashable("was_ecommerce_trial"): false, AnyHashable("is_wpcom_store"): true, AnyHashable("blog_id"): 214354650]`

9. Tap on Set New tax rate. Enable the toggle below to store the selected tax rate.
10. Select a tax rate. The event should be tracked with the right property ("auto_tax_rate_enabled")

`🔵 Tracked tax_rate_selector_tax_rate_tapped, properties: [AnyHashable("was_ecommerce_trial"): false, AnyHashable("auto_tax_rate_enabled"): true, AnyHashable("blog_id"): 214354650, AnyHashable("is_wpcom_store"): true, AnyHashable("plan"): "business-bundle", AnyHashable("site_url"): "https://americanwootester.wpcomstaging.com"]`

## Screenshots



---
